components/navbar/navbar_info_button.jsx

```diff
@@ -36,28 +37,37 @@ export default class NavbarInfoButton extends React.PureComponent {
                     />
                 );
             } else {
-                const link = (
-                    <a
-                        href='#'
-                        onClick={this.showEditChannelHeaderModal}
-                    >
-                        <FormattedMessage
-                            id='navbar.click'
-                            defaultMessage='Click here'
-                        />
-                    </a>
-                );
+                let addOne;
+                if (!this.props.isReadOnly) {
+                    const link = (
+                        <a
+                            href='#'
+                            onClick={this.showEditChannelHeaderModal}
+                        >
+                            <FormattedMessage
+                                id='navbar.click'
+                                defaultMessage='Click here'
+                            />
+                        </a>
+                    );
+                    addOne = (
+                        <div>
```

Comment:
I don't think this div is necessary, but if you want to maintain the original spacing, you could use
```
addOne = (
    <React.Fragment>
        <br/>
        <FormattedMessage .../>
    </React.Fragment>
);
```